main.go

```diff
@@ -109,6 +109,7 @@ func newCORS() *cors.Cors {
 			"Accept-Post",
 			"Connect-Accept-Encoding",
 			"Connect-Content-Encoding",
+			"Connect-Protocol-Version",
```

Comment:
This adds the field name `Connect-Protocol-Version` to the response header `Access-Control-Expose-Headers`. This allows browser clients to access the response header `Connect-Protocol-Version`. But we never send `Connect-Protocol-Version` in responses, right? That means we don't need this change.